QUARKUS-4221.md

```diff
@@ -0,0 +1,75 @@
+# QUARKUS-4221 Productization of Picocli extension
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-4221
+
+PR: https://github.com/quarkus-qe/quarkus-test-suite/pull/2003
+## Scope of the testing
+### What to test
+
+The test coverage will be verified for environments: JVM for Baremetal and Openshift.
```

Comment:
They mention in the guide, that there are some potential problems on Native. Shouldn't we at least minimally check it?

Reply:
I omitted on Native because this support was asked by keycloak and in a previous analysis they said don't worry about native (they don't play with Native executables).

Reply:
Please mention this reasoning in the plan